### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

## ✅ Pre-approval checklist ##

- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

> [!NOTE]
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by dv4harr10 — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

**Review by mcdonnnj — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍

**Review by dv4harr10 — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -173,6 +174,8 @@ jobs:
       ${{ matrix.platform }}-${{ matrix.architecture }}
     needs:
       - diagnostics
+    # This job does not need any permissions
+    permissions: {}
```
Comment (mcdonnnj) [inline .github/workflows/build.yml]: I'm not sure about the output from the monitor Action, but I believe it should have this permission.

```suggestion
    permissions:
      # actions/checkout needs this to fetch code
      contents: read
```

**Review by dav3r — APPROVED** [review]

**Review by dv4harr10 — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml
Comment (mcdonnnj) [inline .github/workflows/codeql-analysis.yml:1]: Please update `crazy-max/ghaction-github-status` to match the version used in `build.yml`. This must've been missed in a previous Lineage PR.

File: .github/workflows/build.yml (hunk 1 above)
Comment (jsf9k) [reply .github/workflows/build.yml]: Please see commit [9326e51](https://github.com/cisagov/skeleton-ansible-role/pull/206/commits/9326e51c1f4e6ff034ff4a548691181c42e6d638).

File: .github/workflows/codeql-analysis.yml
Comment (jsf9k) [reply .github/workflows/codeql-analysis.yml:1]: Please see commit [5ffaf0e](https://github.com/cisagov/skeleton-ansible-role/pull/206/commits/5ffaf0e26eea12fa99dda7f4fbe48e92376831d0).

**Comment by jsf9k** [comment]
The remaining error from the `pip-audit` `pre-commit` hook is resolved by #209.

**Review by dav3r — APPROVED** [review]

**Pull request merged**